### PR TITLE
pkg: fix static check

### PR DESCRIPTION
**Pull request by hwdef**

### CODE REVIEW

File: pkg/drain/cordon.go [hunk 1]
```diff
@@ -62,10 +62,7 @@ func NewCordonHelperFromRuntimeObject(nodeObject runtime.Object, scheme *runtime
 // or false when no change is needed
 func (c *CordonHelper) UpdateIfRequired(desired bool) bool {
 	c.desired = desired
-	if c.node.Spec.Unschedulable == c.desired {
-		return false
-	}
-	return true
+	return !(c.node.Spec.Unschedulable == c.desired)
```
Comment (rifelpet) [inline pkg/drain/cordon.go]:
```suggestion
	return c.node.Spec.Unschedulable != c.desired
```

Comment (hwdef) [reply pkg/drain/cordon.go]: Thanks for your suggestion. I changed it.
Sorry about this mistake.

**Comment by justinsb** [comment]
Thanks @hwdef 

/approve
/lgtm

**Pull request merged**